resources/views/admin/pages/product/form.blade.php

```diff
@@ -0,0 +1,78 @@
+@extends('admin.layouts.main')
+@section('content')
+    <h4 class="mb-4 text-lg font-semibold text-gray-600 dark:text-gray-300">
+        @if (isset($product)) Update @else Add @endif Product
+    </h4>
+    <form action="{{ isset($product) ? route('product.update', $product->id) : route('product.store') }}" method="post" class="mt-5">
+        @csrf
+        <div class="px-4 py-3 mb-8 bg-white rounded-lg shadow-md dark:bg-gray-800">
+            <label class="block text-sm">
+                <span class="text-gray-700 dark:text-gray-400">Name</span>
+                <x-input placeholder="Enter Product Name" name="name" type="text" value="{{ old('name', $product->name ?? '') }}" required />
+                @if($errors->has('name'))
+                <span class="text-xs text-red-600 dark:text-red-400">{{ $errors->first('name') }}</span>
+                @endif
+            </label>
+            <label class="block mt-4 text-sm">
+                <span class="text-gray-700 dark:text-gray-400">Select Brand</span>
+                <select class="block w-full mt-1 text-sm dark:text-gray-300 dark:border-gray-600 dark:bg-gray-700 form-select focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:focus:shadow-outline-gray" name="brand_id" required>
```

Comment:
`required` always set based on table column & validation rule is required or not.